defaults/main.yml

```diff
@@ -1,11 +1,9 @@
 ---
-# default variable values for ansible-role-nessus
-
 # The AWS S3 bucket containing the Nessus Debian package file
 package_bucket: "cisa-cool-third-party-production"
 
 # The name of the Nessus Debian package file
 package_file: "Nessus-{{ version }}-debian6_amd64.deb"
 
 # The version number of the Nessus Debian package file
-version: "8.7.1"
+version: "8.15.4"
```

Comment:
Shall we update to `8.15.5` while we're here since it has come out in the interim?

Reply:
Please see commit 13df473.